charts/eol-exporter/values.yaml

```diff
@@ -0,0 +1,230 @@
+        path: /favicon.ico
+        port: 8080
+      exec: {}
+      tcpSocket: {}
+    livenessProbe:
+      enabled: true
+      failureThreshold: 3
+      periodSeconds: 10
+      successThreshold: 1
+      timeoutSeconds: 1
+      initialDelaySeconds: 10
+      # Must specify either one of the following field when enabled
+      httpGet:
+        path: /favicon.ico
+        port: 8080
+      exec: {}
+      tcpSocket: {}
+    resources:
+      limits:
+        memory: 256Mi
+        cpu: 500m
+      requests:
+        memory: 128Mi
+        cpu: 100m
+    securityContext:
+      runAsNonRoot: true
+      runAsUser: 1002
+      runAsGroup: 1000
+      # fsGroup: 2000
+      # fsGroupChangePolicy: "OnRootMismatch"
+    containerSecurityContext:
+      allowPrivilegeEscalation: false
+      privileged: false
+      runAsNonRoot: true
+      runAsUser: 1002
+      runAsGroup: 1000
+      readOnlyRootFilesystem: false
+      # capabilities:
+      #   drop:
+      #     - ALL
+      #   add:
+      #     - CHOWN
+      #     - NET_ADMIN
+      #     - SYS_TIME
+    strategy:
+      type: RollingUpdate
+      rollingUpdate:
+        maxSurge: 25%
+        maxUnavailable: 25%
+    additionalLabels: {}
+    tolerations: []
+    nodeSelector: {}
+    affinity:
+      nodeAffinity:
+        requiredDuringSchedulingIgnoredDuringExecution:
+          nodeSelectorTerms:
+            - matchExpressions:
+                - key: kubernetes.io/arch
+                  operator: In
+                  values:
+                    - amd64
+    topologySpreadConstraints: {}
+    revisionHistoryLimit: 5
+  additionalContainers: []
+  service:
+    enabled: true
+    annotations: {}
+    ports:
+      - port: 8080
+        name: http
+        protocol: TCP
+        targetPort: 8080
+    type: ClusterIP
+  ingress:
+    enabled: false
+    ingressClassName: nginx
+    servicePort: http
+    pathType: Prefix
+
+    # deviation from standard way of defining the services
+    hosts: []
+    # - host: backend.site.com
+    #   paths:
+    #     - path: /
+    #       pathType: Prefix
+    #       serviceName: backend
+    #       servicePort: http
+    additionalLabels: {}
+    annotations:
+      cert-manager.io/cluster-issuer: letsencrypt-staging
+      nginx.ingress.kubernetes.io/proxy-body-size: 100m
+      nginx.ingress.kubernetes.io/proxy-connect-timeout: '300'
+      nginx.ingress.kubernetes.io/proxy-read-timeout: '300'
+      nginx.ingress.kubernetes.io/server-snippet: |
+        add_header X-Robots-Tag "noindex, nofollow, nosnippet, noarchive";
+
+        # this prevents hidden files (beginning with a period) from being served
+        location ~ /\. {
+          access_log        off;
+          log_not_found     off;
+          deny              all;
+        }
+    tls: []
+    # - hosts:
+    #   - nginx.site.com
+    #   secretName: nginx.site.com-crt
+  pdb:
+    enabled: false
+    minAvailable: 1
+  autoscaling:
```

Comment:
we will not use HPA for this - so don't have it in the values.yaml

Reply:
Removed autostacling